DBZ-8336 Error Handling Mode for ReselectColumnsPostProcessor (#5944)
debezium/debezium: documentation/modules/ROOT/pages/post-processors/reselect-columns.adoc

|6
|Enables or disables reselection based event key field names.

|7

> |Log or fail the connector in  case reselection fail or give empty record.

Reviewer: Log a warning or throw an exception when reselection does not find the row or if a database exception is raised.

For tables that do not have a primary key, set this property to `true`, and configure the `message.key.columns` property in the connector configuration to specify a custom key for the connector to use when it creates events.
The post processor then uses the specified key field names as the primary key in the SQL reselection query.

|[[reselect-columns-post-processor-property-reselect-error-handling-mode]]<<reselect-columns-post-processor-property-reselect-error-handling-mode, `+reselect.error.handling.mode+`>>
|`WARN`

> |Specifies whether the post processor reselects behaviour if reselection fail or give empty record. +

Reviewer: ```suggestion
|Specifies how the reselect post processor reacts to certain conditions. +
```

 +

> By default, it only logs the error in case reselection fail or empty. +

Reviewer: ```suggestion
By default, the post-processor logs a warning message when the row cannot be found or if there is a database selection error. +
```

 +

> You can set this property to `FAIL` to fail the connector in case of reselection fail or empty record.

Reviewer: ```suggestion
When set to `FAIL`, the connector will stop processing changes if the row no longer exists at the time of reselection or if there is a database failure when reselecting the row.
```
